_monitoring-plugins/ad/api.md

```diff
@@ -368,297 +380,1546 @@ POST _plugins/_anomaly_detection/detectors/<detectorId>/_preview
+            "last_update_time": 1633386974533,
+            "indices": [
+              "server_log"
+            ],
+            "window_delay": {
+              "period": {
+                "unit": "Minutes",
+                "interval": 1
+              }
+            },
+            "detection_interval": {
+              "period": {
+                "unit": "Minutes",
+                "interval": 5
+              }
+            },
+            "name": "testhc",
+            "filter_query": {
+              "match_all": {
+                "boost": 1
+              }
+            },
+            "shingle_size": 8,
+            "user": {
+              "backend_roles": [
+                "admin"
+              ],
+              "custom_attribute_names": [],
+              "roles": [
+                "own_index",
+                "all_access"
+              ],
+              "name": "admin",
+              "user_requested_tenant": "__user__"
+            },
+            "detector_type": "MULTI_ENTITY"
+          },
+          "user": {
+            "backend_roles": [
+              "admin"
+            ],
+            "custom_attribute_names": [],
+            "roles": [
+              "own_index",
+              "all_access"
+            ],
+            "name": "admin",
+            "user_requested_tenant": "__user__"
+          }
         }
       }
     ]
   }
 }
 ```
 
-In high cardinality detectors, the result contains entity information.
-
-To see an ordered set of anomaly records for an entity with an anomaly within a certain time range for a specific feature value:
+To search for the latest entity-level task for a high cardinality detector:
 
 #### Request
 
 ```json
-POST _plugins/_anomaly_detection/detectors/results/_search
+GET _plugins/_anomaly_detection/detectors/tasks/_search
+POST _plugins/_anomaly_detection/detectors/tasks/_search
 {
   "query": {
     "bool": {
       "filter": [
         {
           "term": {
-            "detector_id": "4CIGoHUBTpMGN-4KzBQg"
+            "detector_id": "Zi5zTXwBwf_U8gjUTfJG"
           }
         },
         {
-          "range": {
-            "anomaly_grade": {
-              "gt": 0
-            }
+          "term": {
+            "task_type": "HISTORICAL_HC_ENTITY"
           }
         },
         {
-          "nested": {
-            "path": "entity",
-            "query": {
-              "bool": {
-                "must": [
-                  {
-                    "term": {
-                      "entity.value": "i-00f28ec1eb8997685"
-                    }
-                  }
-                ]
-              }
-            }
+          "term": {
+            "is_latest": "true"
           }
         }
       ]
     }
   },
-  "size": 8,
   "sort": [
     {
-      "execution_end_time": {
+      "execution_start_time": {
         "order": "desc"
       }
     }
   ],
-  "track_total_hits": true
+  "size": 100
+}
+```
+
+To search for all entity-level batch task stats:
```

Comment:
a batch task is ambiguous.  Preview API is running batch analysis too.  Also, you are aggregating states.  How about "To search and aggregate states for all entity-level historical tasks:"?